Gemfile

```diff
@@ -4,3 +4,14 @@ git_source(:https_github) { |repo_name| "https://github.com/#{repo_name}" }
 gem 'sqlite3'
 
 gemspec
+
+gem 'appraisal'
```

Comment:
Technically these should all be `require: false` to get the behavior we'd see by including them in the gemspec. It also requires regenerating all of the Appraisal gemfiles, which is why I've been overriding the rule and leaving them in `gemspec` for now, but up to you.